website/content/en/docs/advanced-topics/multi-arch.md

```diff
@@ -36,23 +41,177 @@ docker buildx build --push --platform linux/amd64,linux/arm64,linux/ppc64le,linu
 
-[manifest_list]: https://docs.docker.com/registry/spec/manifest-v2-2/#manifest-list
-[image_index]: https://github.com/opencontainers/image-spec/blob/main/image-index.md
+## Supporting Clusters with Multi-Architecture Compute Nodes
+The [Fundamentals](#Fundamentals) above aim to guide authors on the key steps to building and distributing operators that can run on multiple architectures. These instructions are sufficient when your cluster's compute nodes share the same supported platform. However, operator authors should also understand the implications of running their operators in a cluster with multi-architecture compute nodes, since it is not always guaranteed that the architectures of the compute nodes will match the architectures supported by the operator.
+
+### Safe Scheduling Using Node Affinity
+Node affinity is a mechanism exposed in a Kubernetes pod template that allows a `PodSpec` author to instruct the scheduler to restrict a pod to run only on (or with preference for) nodes that meet specific criteria. In order to ensure that pods are always scheduled to nodes of compatible architecture, it is a best practice for authors to set node affinity requirements to ensure their operators and operands will only schedule to the platforms supported by the images in the pod. If you don't do this, a container scheduled to an incompatible node will immediately crash with an `exec format error`, which will ultimately lead to an `ImagePullBackoff` event as the pod is restarted only to crash again with the same error.
+
+#### Determining the Platforms Supported by an Image
+For a given container image, you can check which platforms are supported by listing them by inspecting the manifest.
+```shell
+docker manifest inspect <image>
+```
+
+Here's an example of platforms distributed for the Alpine Linux container image. Notice that both linux/amd64 and linux/arm64 are supported.
+```shell
+$ docker manifest inspect alpine
+  "schemaVersion": 2,
+   "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
+   "manifests": [
+      {
+         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
+         "size": 528,
+         "digest": "sha256:1304f174557314a7ed9eddb4eab12fed12cb0cd9809e4c28f29af86979a3c870",
+         "platform": {
+            "architecture": "amd64",
+            "os": "linux"
+         }
+      },
+      {
+         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
+         "size": 528,
+         "digest": "sha256:5da989a9a3a08357bc7c00bd46c3ed782e1aeefc833e0049e6834ec1dcad8a42",
+         "platform": {
+            "architecture": "arm",
+            "os": "linux",
+            "variant": "v6"
+         }
+      },
+      ...
+```
+
+If the above command fails, it may be because the referenced name points to a single image rather than a manifest list. In this case, you can find the architecture by inspecting the image.
+```shell
+docker pull <image> && docker inspect <image>
```

Comment:
instead of using docker and asking users to pull the image , we could ask them to `skopeo inspect --raw  <image>` first and if there aren't any manifests in the output just do a `skopeo inspect <image>`  to look at the manifest arch.

Reply:
that sounds cleaner :)
:+1: